cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -247,11 +253,11 @@ def _clear_retire_request():
             )
 
 
-@pytest.mark.meta(coverage=[1695804])
-@pytest.mark.manual
-@pytest.mark.ignore_stream('5.10')
+@pytest.mark.customer_stories
```

Comment:
customer_stories is a requirement marker, and needs to be imported from requirements.py.